pkg/acsengine/defaults.go

```diff
@@ -645,10 +642,12 @@ func setMasterNetworkDefaults(a *api.Properties, isUpgrade bool) {
 	if a.MasterProfile == nil {
 		return
 	}
-
-	// Set default Distro to Ubuntu
-	if a.MasterProfile.Distro == "" {
-		a.MasterProfile.Distro = api.Ubuntu
+	// set default Distro for all what is not OpenShift
```

Comment:
// don't default Distro for OpenShift
(also change comment further down please)